Cargo.toml

```diff
@@ -14,12 +14,12 @@ members = [
   'communities',
   'communities/rpc',
   'communities/rpc/runtime-api',
-  'personhood-oracle',
+  #'personhood-oracle',
```

Comment:
Is this a WIP and you still need to do them?

Reply:
no, @brenzi and i decided to not keep maintaining those pallets as it is not clear if they will be needed in the future.

Reply:
True: https://github.com/encointer/pallets/issues/314